src/tools/fuzzing/fuzzing.cpp

```diff
@@ -609,6 +612,49 @@ void TranslateToFuzzReader::addImportThrowingSupport() {
   wasm.addFunction(std::move(func));
 }
 
+void TranslateToFuzzReader::addImportTableSupport() {
+  // For the table imports to be able to do anything, we must export a table
+  // for them. For simplicity, use the funcref table we use internally, though
+  // we could pick one at random, support non-funcref ones, and even export
+  // multiple ones TODO
+  if (!funcrefTableName) {
+    return;
+  }
+
+  // If a "table" export already exists, skip fuzzing these imports, as the
+  // current export may not contain a valid table for it.
+  if (wasm.getExportOrNull("table")) {
+    return;
+  }
```

Comment:
In which case can this happen?

Reply:
The fuzzer can start with inputs from the test suite and elsewhere, and initial content like that can contain a "table" export. Basically, the fuzzer needs to be able to start from valid wasm file, so we have to handle corner cases like that.